config/settings.yaml.example

```diff
@@ -37,6 +37,11 @@
 #:logging:
 #  :level: debug
 
+# It is also possible to override settings per environment:
+#:logging:
+#  :test:
+#    :pattern: "[%c|%.1l] %m\n"
```

Comment:
Do you mean the logging pattern? That's explained in `logging.yaml` but you are right that is not the best example. I changed it to more obvious one.